package.json

```diff
@@ -4,6 +4,7 @@
   "description": "A Social platform for the creative at heart",
   "main": "index.js",
   "scripts": {
+    "heroku-postbuild": "npm run build",
```

Comment:
Sorry, I may be missing something but I don't see the build script that 'heroku-postbuild' is suppose to run.

Reply:
Also, you can remove the `heroku-postbuild` script because Heroku now runs NodeJS build script by default before pruning the dev dependencies. Please check this article, https://devcenter.heroku.com/changelog-items/1557